src/components/AppWrapper.tsx

```diff
@@ -465,6 +543,34 @@ const AppWrapper: React.FC<Props> = (props: Props): React.ReactElement => {
   >(undefined);
   setConfirmInfo = setConfirmInfoInner;
 
+  // Prompt
+  const [promptInfo, setPromptInfoInner] = useState<
+  | undefined
+  | {
+    title: string;
+    text: string;
+    opts: {
+      placeholder?: string;
+      defaultText?: string;
+      confirmButtonText?: string;
+      confirmButtonVariant?: Variant;
+      cancelButtonText?: string;
+      cancelButtonVariant?: Variant;
+    }
+  }
+  >(undefined);
+  setPromptInfo = setPromptInfoInner;
+
+  // Current prompt input field text
+  const [currentPromptInputFieldText, setCurrentPromptInputFieldText] = useState<string>('');
+
+  // Update input field text when promptInfo changes
+  useEffect(() => {
+    if (promptInfo) {
+      setCurrentPromptInputFieldText(promptInfo.opts.defaultText ?? '');
+    }
+  }, [promptInfo]);
```

Comment:
Instead, could we just add the current prompt input field text to the promptInfo?

```
{
    title: string;
    text: string;
    currentInputFieldText: string,
    opts: {
      placeholder?: string;
      defaultText?: string;
      confirmButtonText?: string;
      confirmButtonVariant?: Variant;
      cancelButtonText?: string;
      cancelButtonVariant?: Variant;
    }
  }
```